bat/tests/09-update-mixver-offline/run.bats

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bats
+
+# shared test functions
+load ../../lib/mixerlib
+
+setup() {
+  global_setup
+}
+
+@test "Create mix 10 with real clear bundle" {
+  mixer-init-stripped-down 25770 10
```

Comment:
should you export the proxies before this, and make this completely offline?

Reply:
It's a commonly used function, I don't want to edit it just to move the proxies above by 1 line, I think this works just fine.

Reply:
Then we should have another explicit bat test making sure the offline capabilities work. In that case we will need to either modify that function or provide a new one.

Reply:
This isn't that test nor the scope/requirement of this test to introduce though... I can add a new function but it's starting to add tests that should be opened in another PR.